CONTRIBUTING.md

```diff
@@ -10,6 +10,17 @@ OPI uses GitHub to manage reviews of pull requests.
 
 * Be sure to sign off on the [DCO](https://github.com/probot/dco#how-it-works).
 
+## Architecture Changes
+
+Changes to project wide goals or scope, subgroup goals or scope, or major
+architectural changes shall be:
+
+* Submitted via PR to the appropriate repo
+* Labeled with the `architecture` label
+* Broadcast to the `opi-dev` mailing list
+* Left open for review for at least 2 days
+* Approved by at least 2 reviewers
```

Comment:
Approved by two maintainers, not reviewers. Reviewers can review, but maintainers need to approve for merge.